modules/nw-sr-iov-network-node-configuration-examples.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-sr-iov-network-node-configuration-examples_{context}"]
+= SR-IOV network node configuration examples
+
+The following example describes the configuration for an InfiniBand device:
+
+.Example configuration for an InfiniBand device
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: <name>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: <sriov_resource_name>
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  numVfs: <num>
+  nicSelector:
+    vendor: "<vendor_code>"
+    deviceID: "<device_id>"
+    rootDevices:
+      - "<pci_bus_id>"
+  linkType: <link_type>
+  isRdma: true
+# ...
+----
+
+The following example describes the configuration for an SR-IOV network device in a {rh-openstack} virtual machine:
+
+.Example configuration for an SR-IOV device in a virtual machine
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: <name>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: <sriov_resource_name>
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  numVfs: 1 <1>
+  nicSelector:
+    vendor: "<vendor_code>"
+    deviceID: "<device_id>"
+    netFilter: "openstack/NetworkID:ea24bd04-8674-4f69-b0ee-fa0b3bd20509" <2>
+# ...
+----
+
+<1> The `numVfs` field is always set to `1` when configuring the node network policy for a virtual machine.
+
+<2> The `netFilter` field must refer to a network ID when the virtual machine is deployed on {rh-openstack}. Valid values for `netFilter` are available from an `SriovNetworkNodeState` object.
```

Comment:
Change to: "When the virtual machine is deployed on {rh-openstack}, the `netFilter` field must refer to a network ID."